### PR TITLE
[Review] Conventions for frontend build tooling

### PR DESCRIPTION
**Pull request by ziahamza**

### CODE REVIEW

**Comment by prayogoa** [comment]
Nice! What about a section on known issues?

File: Build.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The local dev environment is powered through [Create React App](https://github.com/facebookincubator/create-react-app) which simplifies our build toolchain. However, despite using CRA (Create React App), we do not have any strong dependency towards build tools except for CSS modules. We [patch](https://github.com/rainforestapp/regenwald/blob/fc03836cb73421292c8be41620436c220a54efdf/patches/react-scripts%2B1.0.17.patch#L1) CRA at build time to support CSS modules.
+
+## Starting Dev Environment
+All frontned projects should kick start the local dev toolchain when running the following commands
```
Comment (pscohn) [inline Build.md]: frontend

**Comment by jbarber** [comment]
@prayogoa kill or merge?

**Comment by prayogoa** [comment]
kill

**Pull request closed without merging**